get-started/index.md

```diff
@@ -609,25 +609,25 @@ spec:
 ### [Alternate Setup](#alternate-setup)
 
 #### [Using ClusterCryostats](#using-clustercryostats)
-In [Deploying Cryostat](#deploying-cryostat), you created a single-namespace `Cryostat` Custom Resource
-(`CR`) instance.
-
-Single-namespace `Cryostat` `CR`s instruct the Operator to deploy restricted Cryostat instances which are only able
-to see target applications deployed in the same namespace as the Cryostat instance, which is the same Namespace that
-the `CR` is created within.
-
-If you chose to install the Operator in **All Namespaces** mode as assumed in this guide, you may also be interested in
-creating `CluterCryostat` `CR`s. In this configuration, the Operator is able to see `Cryostat` and `ClusterCryostat`
-`CR`s in any project (`Namespace`) and create Cryostat deployments corresponding to either `CR` kind in each of their
-respective `Namespaces`. Both of these `CRs` are `Namespace`-specific, and the `Namespace` is used to determine which
-OpenShift users are able to access the Cryostat instance. For more information, please see the following documents:
+In [Deploying **Cryostat**](#deploying-cryostat), you created a single-namespace **Cryostat** Custom Resource
+(**CR**) instance.
+
+Single-namespace **Cryostat CR\`s** instruct the **Operator** to deploy restricted **Cryostat** instances which are only able
+to see target applications deployed in the same namespace as the **Cryostat** instance, which is the same `Namespace` that
+the **CR** is created within.
+
+If you chose to install the **Operator** in **All Namespaces** mode as assumed in this guide, you may also be interested in
+creating **CluterCryostat CR\`s**. In this configuration, the **Operator** is able to see **Cryostat** and **ClusterCryostat
```

Comment:
Same here, remove "\\`" - this looks like it's a bad "correction" to bold the term while removing the previous monospace formatting.